how2meet/db/models.py

```diff
@@ -28,23 +28,37 @@ class Event(Base):
     organizer_name = Column(String(50))
     organizer_password = Column(String(100))
     description = Column(String(500))
-    invites = relationship("Invite", primaryjoin="Event.id == Invite.event_id", cascade="all, delete-orphan")
+    guests = relationship("Guest", primaryjoin="Event.id == Guest.event_id", cascade="all, delete-orphan")
 
     def __repr__(self):
         return f"Event({self.name}, {self.date})"
 
 
-class Invite(Base):
-    __tablename__ = "invites"
+class Guest(Base):
+    __tablename__ = "guests"
 
     id = Column(String(32), primary_key=True, index=True)
     name = Column(String(100))
-    email = Column(String(100), nullable=True)
+    email = Column(String(100), nullable=True)  # TODO: how to require at least one (email OR phone)? Maybe not what we want anyway
```

Comment:
I don't know that there's value enforcing this at the model/db level? Seems like something we can just require in the frontend when adding a guest.

Reply:
Actually thinking about this more... I don't agree that a guest should require a contact method at all. If we want to follow When2Meet's example of frictionless users, we should allow for a guest to RSVP with no connecting info.